src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SignAndAuthKeys/TokenExpandable.vue

```diff
@@ -240,6 +318,15 @@ export default Vue.extend({
       this.$emit('token-logout');
     },
 
+    tokenNameClick(): void {
+      if (this.isExpanded(this.token.id)) {
+        this.descClose(this.token.id);
+        return;
```

Comment:
I see using return for stopping logic here a bad practise as the content of this function is a regular if-else block

Reply:
Fixed